### PR TITLE
Fix polkadot total balance

### LINKED ISSUE #4799: Displaying wrong total balance in Portfolio for Polkadot
**Describe the bug**

We are not displaying a correct total balance in portfolio for Polkadot addresses that have reserved funds (due to voting on governance).

**Total balance** is the sum of `freeBalance` and `reservedBalance`.

```javascript
const balances = await api.derive.balances.all("15psTaipmWG86U5vNkF7Guv9TRPMRLKHkGS8cXT74v3RCC5t");
console.log(JSON.stringify(balances, null, 2));
```

```json
{
  "accountId": "15psTaipmWG86U5vNkF7Guv9TRPMRLKHkGS8cXT74v3RCC5t",
  "accountNonce": 12,
  "availableBalance": 312795000000,
  "freeBalance": 4312795000000,
  "frozenFee": 4000000000000,
  "frozenMisc": 4000000000000,
  "isVesting": false,
  "lockedBalance": 4000000000000,
  "lockedBreakdown": [
    {
      "id": "0x7374616b696e6720",
      "amount": 4000000000000,
      "reasons": "All"
    },
    {
      "id": "0x706872656c656374",
      "amount": 200000000000,
      "reasons": "Misc"
    },
    {
      "id": "0x64656d6f63726163",
      "amount": 100000000000,
      "reasons": "Misc"
    }
  ],
  "reservedBalance": 206300000000,
  "vestedBalance": 0,
  "vestedClaimable": 0,
  "vestingLocked": 0,
  "vestingTotal": 0,
  "votingBalance": 4312795000000
}
```

**To Reproduce**

1. Go to https://app.lunie.io
2. Login with explore mode using address `15psTaipmWG86U5vNkF7Guv9TRPMRLKHkGS8cXT74v3RCC5t`
2. Compare portfolio balances with Polkadot JS Portal or PolkaScan:

https://polkascan.io/polkadot/account/15psTaipmWG86U5vNkF7Guv9TRPMRLKHkGS8cXT74v3RCC5t

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser: 
 - Version: 

**Additional context**

### PR DESCRIPTION
**Pull request by mariopino**
Closes #4799 

**Description:**

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [x] Attach screenshots of the UI components on the PR description (if applicable)
- [x] Scope of work approved for big PRs

For reviewer:

- [x] Manually tested the changes on the UI

### CODE REVIEW

File: api/lib/source/polkadotV0-source.js [hunk 1]
```diff
@@ -207,8 +207,8 @@ class polkadotAPI {
   async getBalancesV2FromAddress(address, fiatCurrency) {
     const api = await this.getAPI()
     const account = await api.query.system.account(address)
-    const { free, feeFrozen } = account.data.toJSON()
-    const totalBalance = BigNumber(free)
+    const { free, reserved, feeFrozen } = account.data.toJSON()
+    const totalBalance = BigNumber(free).plus(BigNumber(reserved))
```
Comment (faboweb) [inline api/lib/source/polkadotV0-source.js:211]: please comment what reserved is

**Comment by Bitcoinera** [comment]
Do you have any address that serves an example to see this? I had an account with reserved from referendum, but now this referendum ended.

Or maybe just vote with the test account to try this?

**Comment by mariopino** [comment]
> Do you have any address that serves an example to see this? I had an account with reserved from referendum, but now this referendum ended.
> 
> Or maybe just vote with the test account to try this?
> 
> EDIT: `reserved` refers to the locked amount in a referendum proposal, right?

Yep, can also happen if you have an identity set.

Sure,  you can use this polkadot address: 15psTaipmWG86U5vNkF7Guv9TRPMRLKHkGS8cXT74v3RCC5t

**Comment by Bitcoinera** [comment]
Ok, it works. I can confirm :+1:

File: api/lib/source/polkadotV0-source.js [hunk 2]
```diff
@@ -207,6 +207,7 @@ class polkadotAPI {
   async getBalancesV2FromAddress(address, fiatCurrency) {
     const api = await this.getAPI()
     const account = await api.query.system.account(address)
+    // See https://wiki.polkadot.network/docs/en/build-protocol-info#free-vs-reserved-vs-locked-vs-vesting-balance
```
Comment (Bitcoinera) [inline api/lib/source/polkadotV0-source.js:210]: :sweat_smile:  I would include this but also say in plain English: "We need to account for funds that are locked on proposals for voting" or something along those lines

Comment (mariopino) [reply api/lib/source/polkadotV0-source.js:214]: For the lazy (guilty!) I included some considerations about balances :-)

Comment (Bitcoinera) [reply api/lib/source/polkadotV0-source.js:214]: ok nice

**Review by Bitcoinera — APPROVED** [review]
